src/planner.ts

```diff
@@ -178,8 +250,19 @@ class BaseContract {
     });
   }
 
-  static fromEthersContract(contract: EthersContract): Contract {
-    return new Contract(contract.address, contract.interface);
+  static createContract(
+    contract: EthersContract,
+    commandflags = CommandFlags.CALL
+  ): Contract {
+    return new Contract(contract.address, contract.interface, commandflags);
+  }
```

Comment:
I see why its done on the contract level, but wouldn't on the function level be more favourable, I may have a contract where i need to first `staticcall` and then do a `call` based on the returned info.